### DISCUSSION TITLE
embedded image contentful-NextJS

### DISCUSSION THREAD

**Discussion by kev0629** [vercel/next.js#39173] [Help]
Hi all,

I’m using contentful with NextJS with the example given by nextJS. After all setting up creating an article is working well, but when I want to add a embeded assets in my post (an simple image) it's not working. From contentful it working well, no problem to add the embeded asssets in my content, but from my local website, I have the following issue:
<img width="945" alt="Capture d’écran 2022-07-27 à 08 38 35" src="https://user-images.githubusercontent.com/21035698/181808365-1ca2d1ee-cb5d-4f3e-8d4e-650f90e9b6dc.png">

I tried to change some thing in my next.config.js file, but no succes and I have no more idea now...

Some of you were confronted to this problem ?

Thank you

**Comment by Babbili** [comment]
Hi @kev0629 any code to share about the page causing this error ?

**Reply by kev0629** [reply]
I just clone the cms-contentful example from NextJs, I changed nothing on the code.  https://github.com/vercel/next.js/tree/canary/examples/cms-contentful

**Comment by tofuwu-dev** [comment]
Hi @kev0629 
Have you tried to write in next config js file like this:
```typescript
...
images: {
    loader: "imgix",
    path: DOMAIN, // write you production link or localhost:3000 based on your env(for production, development, staging)
    domains: ["images.ctfassets.net"], // write domain which have assets
  },
...
```
Hope to work it out

**Reply by kev0629** [reply]
Hi @haodev007, Thank you for your help, it's almost good, there is no error anymore , but the picture is not displayed... I think we are on the good way !

**Reply by tofuwu-dev** [reply]
Hi @kev0629 
Can you plz the website link you deployed? so I can check something...
I guess, the reason why image is not displayed, it is wrong path on next config js file...

**Reply by kev0629** [reply]
This is what I deploy locally : https://github.com/vercel/next.js/tree/canary/examples/cms-contentful

I write the next.config.js file like this :
`module.exports = {
  images: {
    domains: ["images.ctfassets.net"], // write domain which have assets
  },
}`

Then the picture appear but on the top of my page, I should now find why ...

**Reply by tofuwu-dev** [reply]
I mean your website link which you deployed... like this: https://next-blog-contentful.vercel.app/
It will be better if you can share your code and your env variables, but I am not sure if it is possible...
This example shows SSG of Next.js, so you need to configure properly

**Reply by kev0629** [reply]
My bad, 

Here you can find my verbal link : https://cms-contentful-app-example.vercel.app

and Here my GitHub repo : https://github.com/kev0629/cms-contentful-app


Thank you again for your help 🙏🏼

**Reply by tofuwu-dev** [reply]
> Here you can find my verbal link : https://cms-contentful-app-example.vercel.app
I think, it is working well isnt it? Images looks good

> and Here my GitHub repo : https://github.com/kev0629/cms-contentful-app
This is not accessible for me, unfortunately...

**Reply by kev0629** [reply]
Sorry, the repo is now public.
My issue is now on the Article 2 (https://cms-contentful-app-example.vercel.app/posts/article2)
The bird picture should be on the middle of the text but it's on the top of the page... I think I should investigate on file post-body.js
https://github.com/kev0629/cms-contentful-app/blob/main/components/post-body.js
